words/words.md

```diff
@@ -47,27 +47,27 @@ Also schnall dich an: Wir erwarten einige Turbulenzen.
     </div>
 </div>
 
-Piloten und Pilotinnen benutzen Flugsimulatoren, um zu lernen, wie man fliegt.
+Piloten und Pilotinnen benutzen Flugsimulatoren, um zu lernen, wie man ein Flugzeug wieder heil landet.
 
-**Epidemiologen verwenden Epidemiesimulatoren, um zu lernen, wie man die Menschheit vor dem Aussterben bewahrt.**
+**Epidemiologen verwenden Epidemiesimulatoren, um zu lernen, wie die Menschheit wieder heil landet.**
```

Comment:
"**Epidemiologists use epidemic simulators to learn how not to crash humanity.**"
->
"**Epidemiologen und Epidemiologinnen lernen in Epidemiesimulatoren wie man die Menschheit vor dem Absturz bewahrt.**"

Reply:
Würde ich davon abhängig machen, auf was wir uns in L50 einigen. Bei @herrhundt s Vorschag z.B.:
"Epidemiologinnen und Epidemiologen lernen/trainieren in Epidemiesimulatoren, den Absturz der Menschheit zu verhindern."